composer.json

```diff
@@ -10,22 +10,27 @@
             "name": "Simone Carletti",
             "email": "simone.carletti@dnsimple.com",
             "homepage": "https://developer.dnsimple.com"
+        },
+        {
+            "name": "Enrique Comba Riepenhausen",
+            "email": "enrique@ecomba.pro",
+            "homepage": "http://ecomba.pro"
         }
     ],
     "require": {
-        "php": ">=5.6.0",
-        "guzzlehttp/guzzle": "^6.3",
+        "php": "7.4.7",
```

Comment:
This is too constraining. It's unrealistic to expect such big adoption of the latest version. Most projects are stuck with slow upgrade cycles. I would relax to at least 7.x

Reply:
See https://github.com/dnsimple/dnsimple-php/pull/1#discussion_r448844331 (short version, I'm changing it 😄